sdk/cosmos/azure-cosmos/src/main/java/module-info.java

```diff
@@ -32,7 +32,7 @@
     exports com.azure.cosmos.util;
 
     // exporting some packages specifically for Jackson
-    opens com.azure.cosmos to com.fasterxml.jackson.databind;
+    opens com.azure.cosmos to com.fasterxml.jackson.databind, org.apache.commons.lang3;
```

Comment:
Why do we need this?

Reply:
We use reflection to retrieve `userAgentSuffix` field of cosmosClient: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-spring-data-cosmos-core/src/main/java/com/azure/spring/data/cosmos/CosmosFactory.java#L87

Reply:
But that code piece is in spring-data-cosmos-core and here we are exporting apache commons lang 3 to azure-cosmos module, which doesn't use this dependency at all. Let's discuss this offline today.

Reply:
Yes, cosmos package should have no dependency or reference to `org.apache.commons.lang3`.